contracts/staking/GnosisDepositProcessorL1.sol

```diff
@@ -51,46 +48,27 @@ contract GnosisDepositProcessorL1 is DefaultDepositProcessorL1 {
     function _sendMessage(
         address[] memory targets,
         uint256[] memory stakingIncentives,
-        bytes memory bridgePayload,
+        bytes memory,
         uint256 transferAmount
     ) internal override returns (uint256 sequence) {
-        // Check for the bridge payload length
-        if (bridgePayload.length != BRIDGE_PAYLOAD_LENGTH) {
-            revert IncorrectDataLength(BRIDGE_PAYLOAD_LENGTH, bridgePayload.length);
-        }
-
-        // Transfer OLAS together with message, or just a message
+        // Transfer OLAS tokens
         if (transferAmount > 0) {
             // Approve tokens for the bridge contract
             IToken(olas).approve(l1TokenRelayer, transferAmount);
 
-            bytes memory data = abi.encode(targets, stakingIncentives);
-            IBridge(l1TokenRelayer).relayTokensAndCall(olas, l2TargetDispenser, transferAmount, data);
```

Comment:
We drop the `relayTokensAndCall()` function and send tokens only with the `relayTokens()` one, then deliver each message separately.